src/hotspot/share/runtime/vframe.inline.hpp

```diff
@@ -175,7 +175,7 @@ inline void vframeStreamCommon::fill_from_compiled_frame(int decode_offset) {
       ss.print_cr("Error in fill_from_frame: pc_desc for "
                   INTPTR_FORMAT " not found or invalid at %d",
                   p2i(_frame.pc()), decode_offset);
-      nm()->print_on(&ss);
+      nm()->print_on_v(&ss);
```

Comment:
I suggest removing _v suffix to reduce changes and match existing naming.

Reply:
Done. Testing now.